### PR TITLE
Fix errors detected by PHP_CodeSniffer

### LINKED ISSUE #13181: Some classes and methods doesn't follow the coding standard naming rule
PHP_CodeSniffer has detected errors in some class names and method names.

Classes:
- `PMA\libraries\engines\Mrg_Myisam`
- `PMA\libraries\engines\Performance_Schema`
- `PMA\libraries\plugins\export\PMA_ExportPdf`

Methods:
- `PMA\libraries\PDF::Download`
- `PMA\libraries\PDF::SetAlias`
- `PMA\libraries\plugins\schema\pdf\Pdf::NbLines`
- `PMA\libraries\plugins\schema\pdf\Pdf::Row`
- `PMA\libraries\plugins\schema\pdf\Pdf::SetWidths`
- `PMA\libraries\Response::http_response_code`
- `PMA\libraries\Response::set_http_response_code`

Should these errors be fixed or be ignored?

### PR DESCRIPTION
**Pull request by MauricioFauth**
Fix #13181

Signed-off-by: Maurício Meneghini Fauth <mauriciofauth@gmail.com>

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [x] Any new functionality is covered by tests

### CODE REVIEW

File: libraries/PDF.php [hunk 1]
```diff
@@ -136,7 +136,7 @@ public function Error($error_message = '')
      *
      * @return void
      */
-    public function Download($filename)
+    public function download($filename)
```
Comment (nijel) [inline libraries/PDF.php]: As this class bases on TCPDF, I'd probably keep their coding style here...

File: libraries/plugins/export/ExportPdf.php [hunk 2]
```diff
@@ -12,7 +12,7 @@
 use PMA\libraries\properties\plugins\ExportPluginProperties;
 use PMA\libraries\properties\options\groups\OptionsPropertyMainGroup;
 use PMA\libraries\properties\options\groups\OptionsPropertyRootGroup;
-use PMA\libraries\plugins\export\PMA_ExportPdf;
+use PMA\libraries\plugins\export\PdfExport;
```
Comment (nijel) [inline libraries/plugins/export/ExportPdf.php]: Maybe this class should be placed elsewhere? Having PdfExport end ExportPdf at same level is not really intuitive...

File: libraries/plugins/schema/pdf/Pdf.php [hunk 3]
```diff
@@ -277,7 +277,7 @@ public function Footer()
      *
      * @return void
      */
-    public function SetWidths($w)
+    public function setWidths($w)
```
Comment (nijel) [inline libraries/plugins/schema/pdf/Pdf.php]: Same as with libraries/PDF.php - this extends TCPDF, so maybe it's better to stay consistent within the file?

**Review by nijel — COMMENTED** [review]
Most of the changes make sense, though I'm not really convinced about changing TCPDF subclasses as just moves inconsistency to another level...

File: libraries/PDF.php (hunk 1 above)
Comment (mauriciofauth) [inline libraries/PDF.php]: Coding style is quite inconsistent in TCPDF.
However, it's more common for method names to use the camelCase format. So I switched to that format.

File: libraries/plugins/export/ExportPdf.php (hunk 2 above)
Comment (mauriciofauth) [inline libraries/plugins/export/ExportPdf.php]: Didn't know very well what to do with this file. I also believe that this class should be placed elsewhere, but I do not know where.
Maybe `PMA\libraries\plugins\ExportPdf` or `PMA\libraries\Pdf\PdfExport`.

File: libraries/plugins/schema/pdf/Pdf.php (hunk 3 above)
Comment (mauriciofauth) [inline libraries/plugins/schema/pdf/Pdf.php]: Same as with libraries/PDF.php. Coding style is quite inconsistent in TCPDF. That is why I prefer to follow the PMA standard. What do you think, @nijel?

File: libraries/plugins/export/ExportPdf.php (hunk 2 above)
Comment (mauriciofauth) [inline libraries/plugins/export/ExportPdf.php]: Maybe `PMA\libraries\plugins\helpers\Pdf`

Comment (nijel) [inline libraries/plugins/export/ExportPdf.php]: What do you think about `PMA\libraries\plugins\export\helpers\Pdf`? It's helper for the export plugin in the end...

File: libraries/PDF.php (hunk 1 above)
Comment (nijel) [inline libraries/PDF.php]: You're right, I thought that TCPDF starts methods always with upper case, but that really isn't true these days.

File: libraries/plugins/schema/pdf/Pdf.php (hunk 3 above)
Comment (nijel) [inline libraries/plugins/schema/pdf/Pdf.php]: I agree with your changes in the end, I had false memory of what coding style TCPDF uses...

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**